crates/cli-support/src/intrinsic.rs

```diff
@@ -102,9 +105,75 @@ intrinsics! {
         #[symbol = "__wbindgen_is_string"]
         #[signature = fn(ref_externref()) -> Boolean]
         IsString,
+        #[symbol = "__wbindgen_is_bigint"]
+        #[signature = fn(ref_externref()) -> Boolean]
```

Comment:
Why special-case this? __wbindgen_typeof can be used to implement this.

Reply:
The other types are already special-cased, so I special-cased this one too for consistency